README.md

```diff
@@ -10,6 +10,17 @@ Run the application:
 
 And open it in the browser at [http://127.0.0.1:5000/](http://127.0.0.1:5000/)
 
+## Pre-built Container Images
+
+Cachito container images are automatically built when changes are merged. There are two images,
+an httpd based image with the Cachito API, and a celery work image with the Cachito worker code.
+
+- [![cachito-api](https://quay.io/repository/factory2/cachito-api/status)](https://quay.io/repository/factory2/cachito-api)
```

Comment:
Optional: I think it'd look better without the bullets

Reply:
Yeah, seems nicer that way.